scripts/plugins/utilities.py

```diff
@@ -49,80 +50,77 @@ def __init__(self, name, schema_key, schema_property_values=None):
+        """
+        Visual representation of the schema object parsed.
+        """
+        return f"\n\t\tName:         [{self.name}]\
+            \n\t\tSchema Key:   [{self.schema_key}]\
+            \n\t\tConfig_Key:   [{self.config_key}]\
+            \n\t\tSchema Type:  [{self.schema_property_type}]\
             \n                      \
-            \n\t\tExport Env:   [{}]\
-            \n\t\tDefault:      [{}]\
-            \n\t\tEnabled:      [{}]\
-            \n\t\tType:         [{}]\
-            \n\t\tParameter:    [{}]\
-            \n\t\tDash Type:    [{}]\
-            \n\t\tRequired:     [{}]\
+            \n\t\tExport Env:   [{self.export_env}]\
+            \n\t\tDefault:      [{self.default}]\
+            \n\t\tEnabled:      [{self.enabled}]\
+            \n\t\tType:         [{self.type}]\
+            \n\t\tParameter:    [{self.parameter}]\
+            \n\t\tDash Type:    [{self.dash_type}]\
+            \n\t\tRequired:     [{self.required}]\
             \n                      \
-            \n\t\tValue Set:    [{}]".format(
-            self.name,
-            self.schema_key,
-            self.config_key,
-            self.schema_property_type,
-            self.export_env,
-            self.default,
-            self.enabled,
-            self.type,
-            self.parameter,
-            self.dash_type,
-            self.required,
-            self.value,
-        )
-
-    def ProcessConfig(self, config_yaml):
+            \n\t\tValue Set:    [{self.value}]"
+
+    def process_config(self, config_yaml):
+        """
+        Function that processes the bitops.config against the SchemaObject,
+        with the defaults loaded from the bitops.schema.
+        It also checks that the type for the SchemaObject matches that in the configuration.
+        """
         if self.type == "object":
             return
-        result = Get_Nested_Item(config_yaml, self.config_key)
-        logger.info(
-            "\n\tSearching for: [{}]\n\t\tResult Found: [{}]".format(
-                self.config_key, result
-            )
-        )
-        found_config_value = Apply_Data_Type(self.type, result)
+        result = get_nested_item(config_yaml, self.config_key)
+        logger.info(f"\n\tSearching for: [{self.config_key}]\n\t\tResult Found: [{result}]")
+        found_config_value = apply_data_type(self.type, result)
 
         if found_config_value:
             logger.info(
-                "Override found for: [{}], default: [{}], new value: [{}]".format(
-                    self.name, self.default, found_config_value
-                )
+                f"Override found for: [{self.name}], default: [{self.default}], "
+                f"new value: [{found_config_value}]"
             )
             self.value = found_config_value
         else:
             self.value = self.default
 
-        AddValueToEnv(self.export_env, self.value)
+        add_value_to_env(self.export_env, self.value)
```

Comment:
There was a standard being followed in regards to naming. 
- Variables use lowercase_snakecase
- Function Names were using UpperCase_SnakeCase
- Class Names were using UpperCamalCase

I'd appreciate a discussion before changing function names as you plan here.

Reply:
Though i will admit, looking through my standard wasn't exactly followed consistently.

Reply:
Pylint just enforces the official python PEP8 (https://peps.python.org/pep-0008/) guidelines by the python community.
* `snake_case` is used for functions, methods, and variables
* `PascalCase` is used for classes
* `UPPER_CASE` is used for constants

It's best for the project to follow the same standards.
https://pylint.pycqa.org/en/latest/user_guide/messages/convention/invalid-name.html

Here is the commit that aggregates these changes to see the relevant diff: https://github.com/bitovi/bitops/pull/298/commits/48f7222a68247297876a670ae925a3402f0723b8

Reply:
Fair enough